packages/blade/rollup.config.mjs

```diff
@@ -51,10 +52,19 @@ const themeBundleCategories = ['tokens', 'utils'];
 
 const aliases = pluginAlias({
   entries: [
-    { find: '~src', replacement: `${__dirname}/${inputRootDirectory}` },
-    { find: '~components', replacement: `${__dirname}/${inputRootDirectory}/components` },
-    { find: '~utils', replacement: `${__dirname}/${inputRootDirectory}/utils` },
-    { find: '~tokens', replacement: `${__dirname}/${inputRootDirectory}/tokens` },
+    { find: '~src', replacement: fileURLToPath(new URL(inputRootDirectory, import.meta.url)) },
```

Comment:
What is this fileURLToPath doing, and can't this be done with the node:path module? 🤔

Reply:
> can't this be done with the node:path module?

No, this is also [recommended from rollup.](https://rollupjs.org/command-line-interface/#caveats-when-using-native-node-es-modules)